content/community/telegram_orchestrator_watcher_bot.md

```diff
@@ -0,0 +1,8 @@
+---
+image: >-
+  https://res.cloudinary.com/duquzk7m6/image/upload/v1665429174/telegram-orchestrator-watcher-bot_swkovm.png
+title: Telegram Orchestrator Watcher Bot
```

Comment:
Could we fix the line height on this and make it spaced better?

![Uploading Screen Shot 2022-10-11 at 11.57.57 AM.png…]()

Reply:
Sorted